user-guide/Advanced_Functionality/DataMiner_Systems/Adding_a_DMA/Adding_a_DMA_to_a_DMS_running_STaaS.md

```diff
@@ -4,51 +4,54 @@ uid: Adding_a_DMA_to_a_DMS_running_STaaS
 
 # Adding a DataMiner Agent to a DMS running STaaS
 
-Follow these steps below to connect a DMA to an existing DMS running STaaS:
+To add a DataMiner Agent to a DMS running STaaS:
 
-1. Ensure the DMA you are adding is online and empty.
+1. Make sure that the DMA you are adding is a clean DMA, meaning that it is a newly installed DMA or a DMA that has been [fully reset](xref:Factory_reset_tool).
 
-1. Disconnect the DMA you are adding from dataminer.services.
+   > [!IMPORTANT]
+   > If the CloudGateway DxM is installed on the DMA you are adding, the data folder `C:\ProgramData\Skyline Communications\DataMiner CloudGateway\Data` must be empty.
 
-   To make sure the DMA is disconnected, follow the procedure on [Permanently disconnecting from dataminer.services](xref:Disconnecting_from_dataminer.services#permanently-disconnecting-from-dataminerservices).
+1. Stop the DMA that will be added.
 
-   > [!IMPORTANT]
-   > Only disconnect the DMA you are adding, NOT the cluster you are adding it to.
+1. On the DMA that will be added, stop CloudGateway if it is installed.
+
+1. Go to the `C:\Skyline DataMiner\` folder of an existing DMA in the cluster, and follow the steps below:
+
+   1. Copy the `C:\Skyline DataMiner\NATS\nsc` folder and place it in the same location on the new DMA.
 
-1. Make sure STaaS is not enabled in *DB.xml* for the DMA you are adding:
+   1. Copy the `C:\Skyline DataMiner\SLCloud.xml` file to the same location on the new DMA.
 
-   1. Stop the DataMiner Agent.
+1. Configure the new DMA:
 
-   1. In the `C:\Skyline DataMiner` folder of the DMA, open the file *DB.xml*.
+   1. Open `C:\Skyline DataMiner\SLCloud.xml` and modify the `<ClientID>` to match the name of the server.
 
-   1. Make sure that the *Database* tag with *type="CloudStorage"* has the *active* attribute set to **false**:
+   1. Open `C:\Skyline DataMiner\MaintenanceSettings.xml` and add the following tag under the `<SLNet>` section:
 
       ```xml
-      <?xml version="1.0"?>
-      <DataBases xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns="http://www.skyline.be/config/db">
-         <DataBase active="false" local="true" search="true" cloud="true" type="CloudStorage"/>
-      </DataBases>
+      <NATSForceManualConfig>true</NATSForceManualConfig>
```

Comment:
I hadn't noticed this earlier, but [elsewhere in the help](https://docs.dataminer.services/user-guide/Reference/DataMiner_Tools/SLNetClientTest_tool/SLNetClientTest_tool_advanced_procedures/SLNetClientTest_disabling_automatic_nats_config.html) we mention that setting NATSForceManualConfig to true has some far-reaching consequences, as this means the user will be responsible for manually maintaining several config files as well as the synchronization of credentials. If this is indeed also the case with this configuration, we should probably add a note and a link to that page with more info on this configuration.

Reply:
We are removing the tag later on in the last step, but adding a warning here should ensure that people will certainly remove the step later on.

Reply:
Added a note with more info to make this clear 😊